### PR TITLE
Added API fetch request

### PR DESCRIPTION
**Pull request by parkeralexjm**
- Deleted header file
- Added API fetch to homepage, should display a random game from board game atlas
- Streamlined href from list array

### CODE REVIEW

File: src/components/collection.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  if (error) return <div>{error.message}</div>
+
+  // If logged in then display a welcome message otherwise prompt user to login
+  const renderCollection = () => {
+    if (user) {
+      return <h2>This is where the persons collection will display</h2>
+    } else {
+      return <h2>Please login to your account</h2>
+    }
+  }
```
Comment (sis) [inline src/components/collection.tsx:17]: I wouldn't let them get to this page without being logged in at all as I don't see a point

You could setup a redirect if there is no user present.

File: src/components/collection.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { Box } from '@mui/material';
+import { useUser } from '@auth0/nextjs-auth0/client';
+
+const Collection = () => {
+
+  const { user, error, isLoading } = useUser();
+
+  // Make a loading component later to make this more interesting
+  if (isLoading) return <div>Loading...</div>
+  if (error) return <div>{error.message}</div>
```
Comment (sis) [inline src/components/collection.tsx]: I don't think you should show an error on this component for a general authentication error

File: src/components/homepage.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+type Game = { name:string}
+
+const Homepage = () => {
+  const [randomGame, setRandomGame] = useState<{ count: number; games: Game[] }>()
+  useEffect(() => {
+    fetch('https://api.boardgameatlas.com/api/search?random=true&limit=1&client_id=p5N6VqtA3g')
+      .then(res => res.json())
+      .then(data => setRandomGame(data));
+  }, []);
```
Comment (sis) [inline src/components/homepage.tsx:14]: There are nicer ways to handle fetching from apis but for now this is fine

File: src/components/homepage.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+  // Make a loading component later to make this more interesting and reposition to the correct div
+  const { user, error, isLoading } = useUser();
+  if (isLoading) return <div>Loading...</div>
+  if (error) return <div>{error.message}</div>
```
Comment (sis) [inline src/components/homepage.tsx]: Again, not a fan of these here.

File: src/components/collection.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  const renderCollection = () => {
+    if (user) {
+      return <h2>This is where the persons collection will display</h2>
+    } else {
+      return <h2>Please login to your account</h2>
+    }
+  }
+
+  return (  
+      <Box sx={{ width: "100vw", height: "100vh", backgroundColor: '#012835', pt:'64px', display:'flex', flexDirection: 'column'}}>
```
Comment (sis) [inline src/components/collection.tsx:20]: All this custom inline styling is going to be a pain to maintain longterm but if the aim is to get stuff on the page and deal with that later then fine!

File: .eslintrc.json [hunk 6]
```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": "next/core-web-vitals",
+  "rules": {
+    // ESLint is throwing errors with using <a> tags for routing to the login page, this removes the error
+    "@next/next/no-html-link-for-pages": "off"
+  }
```
Comment (sis) [inline .eslintrc.json:6]: I can copy and paste what we use at my work if you want for Eslint, it's noice

**Review by sis — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 6 above)
Comment (parkeralexjm) [reply .eslintrc.json:6]: Sounds good!

File: src/components/collection.tsx (hunk 5 above)
Comment (parkeralexjm) [reply src/components/collection.tsx:20]: Yeah really it was just so i could see roughly what it would look like

**Review by sis — APPROVED** [review]
Some things I don't like but i dont want to nitpick too much

**Pull request merged**